wiki/Bazel-Setup-Instructions-for-Mac.md

```diff
@@ -0,0 +1,105 @@
+## Bazel Set up for Mac including Apple silicon (M1/M2) chips
+
+## Instructions
+
+**The steps to install Bazel on Mac are:**
+1. Set up Rosetta Terminal
+2. Install Bazel 
+3. Install OpenJDK 8
+4. Install Python 2 and make sure it is active in your environment
+5. Set Bazel, Python 2, ANDROID_HOME paths permanently in your terminal
+6. Prepare the build environment
+7. Verify that the build is working
+
+### 1. Set up Rosetta Terminal
+
+- In the Finder app on your Mac, locate the Applications folder from the favorites sidebar.
+- Right-click on your Terminal app and create a duplicate Terminal (and rename it accordingly, say **Terminal Rosetta**, to avoid confusion).
+- On the newly created Terminal Rosetta icon, right-click and select "Get info", and under “General”, check the option "Open using Rosetta".
+
+**Note: Always use the Rosetta terminal for Bazel setup and running `bash setup.sh` or any Bazel build-related commands.**
+
+### 2. Install Bazel
+
+1. Install Bazel following the instructions [here](https://docs.bazel.build/versions/4.0.0/install-os-x.html#install-with-installer-mac-os-x). Make sure that you follow the instructions for installing a specific version (Oppia Android requires 4.0.0 and won't build on other versions). 
+
+2. That’s it, now Bazel is install, you can check it by running the command:
+   ```
+   bazel --version
+   ```
+   - **Expected Output**
+   ```
+   bazel 4.0.0
+   ```
+
+### 3. Install OpenJDK 8
+
+Oppia Android also requires OpenJDK 8.
+Follow the instructions [here](https://installvirtual.com/install-openjdk-8-on-mac-using-brew-adoptopenjdk/) to install OpenJDK 8. 
+Note that this requires the installation of brew as a pre-requisite, which can be done by following the instructions [here](https://mac.install.guide/homebrew/index.html). You can then set up your `$JAVA_HOME` environment variable using these [instructions](https://stackoverflow.com/a/75167958/11396524).
+
+
+### 4. Install Python 2 and make sure it is active in your environment
+
+To install Python 2 in MacOS follow the follows the commands given below. Note that, this requires installation of brew as a pre-requisite, which can be done by following the instructions [here](https://mac.install.guide/homebrew/index.html). 
+```
+brew install pyenv
+pyenv install 2.7.18
+pyenv global 2.7.18
+```
+
+- To make sure Python 2 is active in your environment, navigate to the **oppia-android** directory and run the following commands before running the Bazel build.
```

Comment:
Isn't this an environment thing that should be done only in step 5?

Reply:
Yes I also think that. Thanks updating.